### ISSUE TITLE
Design Update:

### ISSUE THREAD

**Issue by ghost** [5CS024-Team1/asset-tracker-web#27]
- [x] Add NHS logo to site

**Comment by LuceMook87** [comment]
After doing some research on the way the NHS present their UI's it might be beneficial to adopt a similar layout and scheme to our UI to add uniformity to the product?

https://www.nhs.uk/Services/hospitals/Services/DefaultView.aspx?id=1082

**Comment by MandyMoo3677** [comment]
I dont mind being overuled but would the below be a quick fix on this?  If it causes to much issue, please ignore it.
![AA logo](https://user-images.githubusercontent.com/61636956/80191927-54e24300-860e-11ea-963f-c40376e1fc2a.png)

**Comment by ghost** [comment]
> I dont mind being overuled but would the below be a quick fix on this? If it causes to much issue, please ignore it.

Yes that looks good Mandy i'll see if I can switch the logo to this one intead today

**Comment by 1827159** [comment]
I like this version a lot!

**Comment by Philwlv** [comment]
I Love It 😄

**Comment by ghost** [comment]
Updated Logo on the dev version - 3d081de8edf750d25de80499e08d7fe1d964d979
